### DISCUSSION TITLE
Grep not working as expected. (Global File Search in Project)

### DISCUSSION THREAD

**Discussion by ranveersequeira** [LunarVim/LunarVim#4028] [Q&A] [answered]
I have opened a Project in lvim, I'm trying to global search a text to find files which contains the same text. Just like we do in VSCode Global Search. I'm trying cntr+s+t 
![image](https://user-images.githubusercontent.com/26397867/231370702-8ac0c662-1025-4adc-9b1e-569e3fe332e1.png)
But it not showing anything even after searching for anything. What I'm doing wrong here?

**Comment by LostInTheLogs** [comment, marked as answer]
do you have ripgrep installed?, see :messages

**Reply by ranveersequeira** [reply]
Thanks a lot, I just installed ripgrep. It is working fine now.
